### ISSUE TITLE
FR: Unclack/Knatter

### ISSUE THREAD

**Issue by AtmanActive** [mumble-voip/mumble#6083]
### Context

Voice communication while gaming.

### Description

There is a genious way to filter-out keyboard and/or mouse clicking sounds from audio.

It is something we would call type-to-mute, or short-bursted-mutes-on-click, or similar, but since programs that do this already exist, then, for now, I will refer to it as [Unclack](https://unclack.app) / [Knatter](https://github.com/andersforsgren/knatter) .

Generally speaking, it is a Push-to-Mute special mode, bound to all keypresses (optionally all mouse buttons as well) with the following distinction:
- mute is executed only once, in a short burst on keydown and is not held while the button is held. The release time should be user-configurable, usually ranging from 100ms to 700ms depending on room environment.

As I was testing [Knatter](https://github.com/andersforsgren/knatter) on windows, I was blown away how good this approach works, and wanted to suggest the same functionality to be built in to Mumble, as an optional feature, of course.

There are so many situations where Mumble's Voice-Activated mode works great, but it gets disturbed by loud keyboard and mouse-clicking noises. Unclack/Knatter is a perfect remedy for those situations.

Of course, this feature, if ever built in, should work in conjuction with Voice-Activated and Continuous modes, while it doesn't really make sense with Push-To-Talk mode.

Thanks.

### Mumble component

Client

### OS-specific?

No

**Comment by Krzmbrzl** [comment]
Yeah, this is indeed a duplicate of #1864 and as I have written there, attempting to bind to any key that may potentially make a sound when activated, seems futile and also not desirable. 

Especially since shortcut APIs seem to tend to go towards not allowing global input monitoring but rather require specific shortcuts to be registered with e.g. the desktop environment (that's what Wayland requires).

Imo something like this should be implemented by a separate tool that interfaces directly with the OS. Doing it inside an application (that must work on different OS with different desktop environments and display managers) just seems too much overhead.

**Comment by AtmanActive** [comment]
How about adding this special Push-To-Mute mode (keydown only with configurable release time) to Shortcuts so we can manually configure which keys to unclack?
